### DISCUSSION TITLE
its possible that the purchase?.transactionDate return string!!

### DISCUSSION THREAD

**Discussion by carlosandres555** [hyochan/react-native-iap#2390] [Q&A]
hey, i have a weird case,  somes user return string in transactionDate, its possible, normaly should be return a number!!  and when use this funtion generate invalidate Date because the transaction is string!! 

 let startDate = new Date(purchase?.transactionDate);

Maybe a bug?

**Comment by andresesfm** [comment]
It looks like for Android is a number and for iOS is a string
